src/elements/content-uploader/UploadsManager.scss

```diff
@@ -29,7 +29,7 @@
         display: none;
         height: 375px;
 
-        .buik-btn-plain:focus {
+        .buik-bdl-PlainButton:focus {
```

Comment:
is `buik` another prefix?

Reply:
This file appears to belong to a UI that is separate from the regular Content Uploader. I will revert this change.